maps/map_files/Ice_Colony_v2/Ice_Colony_v2.dmm

```diff
@@ -853,7 +853,6 @@
 	dir = 4
 	},
 /obj/structure/machinery/door/airlock/almayer/secure/colony{
-	locked = 0;
```

Comment:
There are several instances where this happened. 
I have no idea what caused it and I can't seem to find the blame for the commit on github.com

I thought it might be from a merge but I don't see a merge in the commits....

Reply:
there doesn't seem to be a button to "revert change on line" on github on firefox.

Reply:
these happen because `locked` defaults to `0` on `/obj/structure/machinery/door/airlock` and no child type overrides it